index.html

```diff
@@ -34,6 +34,187 @@ <h2>I'm a Software Developer</h2>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups 
+              required. Has been the industry's standard.
+            </p>
+            <ul class="list-x">
+              <li><a class="start-list-x" href="#">HTML</a></li>
+              <li><a class="second-list-x" href="#">CSS</a></li>
+              <li><a class="third-list-x" href="#">Javascript</a></li>
+            </ul>
+            <button>See Project</button>
+          </section>
+
+          <section class="print-data">
+            <h3>Profesional Art Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups 
+              required. Has been the industry's standard.
+            </p>
+            <ul class="list-x">
+              <li><a class="start-list-x" href="#">HTML</a></li>
+              <li><a class="second-list-x" href="#">CSS</a></li>
+              <li><a class="third-list-x" href="#">Javascript</a></li>
+            </ul>
+            <button>See Project</button>
+          </section>
+          
+          <section class="print-data">
+            <h3>Profesional Art Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups 
+              required. Has been the industry's standard.
+            </p>
+            <ul class="list-x">
+              <li><a class="start-list-x" href="#">HTML</a></li>
+              <li><a class="second-list-x" href="#">CSS</a></li>
+              <li><a class="third-list-x" href="#">Javascript</a></li>
+            </ul>
+            <button>See Project</button>
+          </section>
+
+          <section class="print-data">
+            <h3>Profesional Art Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups 
+              required. Has been the industry's standard.
+            </p>
+            <ul class="list-x">
+              <li><a class="start-list-x" href="#">HTML</a></li>
+              <li><a class="second-list-x" href="#">CSS</a></li>
+              <li><a class="third-list-x" href="#">Javascript</a></li>
+            </ul>
+            <button>See Project</button>
+          </section>
+
+          <section class="print-data">
+            <h3>Profesional Art Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups 
+              required. Has been the industry's standard.
+            </p>
+            <ul class="list-x">
+              <li><a class="start-list-x" href="#">HTML</a></li>
+              <li><a class="second-list-x" href="#">CSS</a></li>
+              <li><a class="third-list-x" href="#">Javascript</a></li>
+            </ul>
+            <button>See Project</button>
+          </section>
+
+          <section class="print-data">
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 for each card we could use the article tag instead of the section tag for the best practices of HTML & CSS.